ci-operator/step-registry/openstack-k8s-operators/test/build-deploy-tempest/openstack-k8s-operators-test-build-deploy-tempest-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: openstack-k8s-operators-test-build-deploy-tempest
+  steps:
+    post:
+    - ref: openstack-k8s-operators-gather
```

Comment:
this will gather the operator logs but we don't have openstack service log there yet so if a tempest test fails then it might be hard to know what happened.

Reply:
Removing

Reply:
I think @gibizer meant we need to create step to collect logs from services. @gibizer could you please confirm.

Reply:
We definitely need openstack-k8s-operators-gather to be able to troubleshoot a deployment issue. But yes, I'd also like to get enough logs from the openstack services to be able to troubleshoot a tempest test failure.